### DISCUSSION TITLE
java.lang.RuntimeException: Could not execute entrypoint stage 'main' due to errors, provided by 'emotecraft'!

### DISCUSSION THREAD

**Discussion by Sleeper77713** [FabricMC/fabric-api#2990] [Player support] [answered]
---- Minecraft Crash Report ----
// Daisy, daisy...

Time: 2023-04-02 18:08:23
Description: Initializing game

java.lang.RuntimeException: Could not execute entrypoint stage 'main' due to errors, provided by 'emotecraft'!
	at net.fabricmc.loader.impl.entrypoint.EntrypointUtils.lambda$invoke0$0(EntrypointUtils.java:51)
	at net.fabricmc.loader.impl.util.ExceptionUtil.gatherExceptions(ExceptionUtil.java:33)
	at net.fabricmc.loader.impl.entrypoint.EntrypointUtils.invoke0(EntrypointUtils.java:49)
	at net.fabricmc.loader.impl.entrypoint.EntrypointUtils.invoke(EntrypointUtils.java:35)
	at net.fabricmc.loader.impl.game.minecraft.Hooks.startClient(Hooks.java:52)
	at net.minecraft.class_310.<init>(class_310.java:459)
	at net.minecraft.client.main.Main.method_44604(Main.java:205)
	at net.minecraft.client.main.Main.main(Main.java:51)
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:462)
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:74)
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23)
Caused by: java.lang.NullPointerException: Cannot read field "emotesDir" because "io.github.kosmx.emotes.executor.EmoteInstance.config" is null
	at io.github.kosmx.emotes.executor.EmoteInstance.getExternalEmoteDir(EmoteInstance.java:32)
	at io.github.kosmx.emotes.main.MainLoader.main(MainLoader.java:30)
	at io.github.kosmx.emotes.fabric.FabricWrapper.onInitialize(FabricWrapper.java:28)
	at net.fabricmc.loader.impl.entrypoint.EntrypointUtils.invoke0(EntrypointUtils.java:47)
	... 8 more

A detailed walkthrough of the error, its code path and all known details is as follows:
---------------------------------------------------------------------------------------

-- Head --
Thread: Render thread
Stacktrace:
	at net.fabricmc.loader.impl.entrypoint.EntrypointUtils.lambda$invoke0$0(EntrypointUtils.java:51)
	at net.fabricmc.loader.impl.util.ExceptionUtil.gatherExceptions(ExceptionUtil.java:33)
	at net.fabricmc.loader.impl.entrypoint.EntrypointUtils.invoke0(EntrypointUtils.java:49)
	at net.fabricmc.loader.impl.entrypoint.EntrypointUtils.invoke(EntrypointUtils.java:35)
	at net.fabricmc.loader.impl.game.minecraft.Hooks.startClient(Hooks.java:52)
	at net.minecraft.class_310.<init>(class_310.java:459)

-- Initialization --
Details:
	Modules: 
		ADVAPI32.dll:Расширенная библиотека API Windows 32:10.0.19041.2604 (WinBuild.160101.0800):Microsoft Corporation
		COMCTL32.dll:Библиотека элементов управления взаимодействия с пользователем:6.10 (WinBuild.160101.0800):Microsoft Corporation
		CRYPT32.dll:API32 криптографии:10.0.19041.2604 (WinBuild.160101.0800):Microsoft Corporation
		CRYPTBASE.dll:Base cryptographic API DLL:10.0.19041.546 (WinBuild.160101.0800):Microsoft Corporation
		CRYPTSP.dll:Cryptographic Service Provider API:10.0.19041.546 (WinBuild.160101.0800):Microsoft Corporation
		DBGHELP.DLL:Windows Image Helper:10.0.19041.867 (WinBuild.160101.0800):Microsoft Corporation
		DNSAPI.dll:Динамическая библиотека API DNS-клиента:10.0.19041.2604 (WinBuild.160101.0800):Microsoft Corporation
		GDI32.dll:GDI Client DLL:10.0.19041.746 (WinBuild.160101.0800):Microsoft Corporation
		IMM32.DLL:Multi-User Windows IMM32 API Client DLL:10.0.19041.546 (WinBuild.160101.0800):Microsoft Corporation
		IPHLPAPI.DLL:API вспомогательного приложения IP:10.0.19041.2604 (WinBuild.160101.0800):Microsoft Corporation
		KERNEL32.DLL:Библиотека клиента Windows NT BASE API:10.0.19041.804 (WinBuild.160101.0800):Microsoft Corporation
		KERNELBASE.dll:Библиотека клиента Windows NT BASE API:10.0.19041.804 (WinBuild.160101.0800):Microsoft Corporation
		NLAapi.dll:Network Location Awareness 2:10.0.19041.789 (WinBuild.160101.0800):Microsoft Corporation
		NSI.dll:NSI User-mode interface DLL:10.0.19041.610 (WinBuild.160101.0800):Microsoft Corporation
		NTASN1.dll:Microsoft ASN.1 API:10.0.19041.1 (WinBuild.160101.0800):Microsoft Corporation
		Ole32.dll:Microsoft OLE для Windows:10.0.19041.1 (WinBuild.160101.0800):Microsoft Corporation
		OleAut32.dll:OLEAUT32.DLL:10.0.19041.804 (WinBuild.160101.0800):Microsoft Corporation
		PSAPI.DLL:Process Status Helper:10.0.19041.546 (WinBuild.160101.0800):Microsoft Corporation
		Pdh.dll:Модуль поддержки данных производительности Windows:10.0.19041.2604 (WinBuild.160101.0800):Microsoft Corporation
		RPCRT4.dll:Библиотека удаленного вызова процедур:10.0.19041.2604 (WinBuild.160101.0800):Microsoft Corporation
		SHCORE.dll:SHCORE:10.0.19041.2604 (WinBuild.160101.0800):Microsoft Corporation
		SHELL32.dll:Общая библиотека оболочки Windows:10.0.19041.2604 (WinBuild.160101.0800):Microsoft Corporation
		USER32.dll:Многопользовательская библиотека клиента USER API Windows:10.0.19041.2604 (WinBuild.160101.0800):Microsoft Corporation
		USERENV.dll:Userenv:10.0.19041.2604 (WinBuild.160101.0800):Microsoft Corporation
		VCRUNTIME140.dll:Microsoft® C Runtime Library:14.29.30139.0 built by: vcwrkspc:Microsoft Corporation
		VERSION.dll:Version Checking and File Installation Libraries:10.0.19041.546 (WinBuild.160101.0800):Microsoft Corporation
		WINHTTP.dll:Службы HTTP Windows:10.0.19041.1 (WinBuild.160101.0800):Microsoft Corporation
		WINMM.dll:MCI API DLL:10.0.19041.2604 (WinBuild.160101.0800):Microsoft Corporation
		WS2_32.dll:32-разрядная библиотека Windows Socket 2.0:10.0.19041.1 (WinBuild.160101.0800):Microsoft Corporation
		WSOCK32.dll:Windows Socket 32-Bit DLL:10.0.19041.2604 (WinBuild.160101.0800):Microsoft Corporation
		Wldp.dll:Политика блокировки Windows:10.0.19041.2604 (WinBuild.160101.0800):Microsoft Corporation
		amsi.dll:Anti-Malware Scan Interface:10.0.19041.746 (WinBuild.160101.0800):Microsoft Corporation
		antimalware_provider.dll:Kaspersky AntiMalwareProvider Component:30.587.0.880:AO Kaspersky Lab
		bcrypt.dll:Библиотека криптографических примитивов Windows:10.0.19041.2604 (WinBuild.160101.0800):Microsoft Corporation
		bcryptPrimitives.dll:Windows Cryptographic Primitives Library:10.0.19041.662 (WinBuild.160101.0800):Microsoft Corporation
		clbcatq.dll:COM+ Configuration Catalog:2001.12.10941.16384 (WinBuild.160101.0800):Microsoft Corporation
		combase.dll:Microsoft COM для Windows:10.0.19041.1 (WinBuild.160101.0800):Microsoft Corporation
		dbgcore.DLL:Windows Core Debugging Helpers:10.0.19041.789 (WinBuild.160101.0800):Microsoft Corporation
		dhcpcsvc.DLL:Служба DHCP-клиента:10.0.19041.2604 (WinBuild.160101.0800):Microsoft Corporation
		dhcpcsvc6.DLL:Клиент DHCPv6:10.0.19041.2604 (WinBuild.160101.0800):Microsoft Corporation
		fwpuclnt.dll:API пользовательского режима FWP/IPsec:10.0.19041.2604 (WinBuild.160101.0800):Microsoft Corporation
		gdi32full.dll:GDI Client DLL:10.0.19041.746 (WinBuild.160101.0800):Microsoft Corporation
		glfw.dll:GLFW 3.4.0 DLL:3.4.0:GLFW
		java.dll:OpenJDK Platform binary:17.0.3.0:Microsoft
		javaw.exe:OpenJDK Platform binary:17.0.3.0:Microsoft
		jemalloc.dll
		jimage.dll:OpenJDK Platform binary:17.0.3.0:Microsoft
		jli.dll:OpenJDK Platform binary:17.0.3.0:Microsoft
		jna8137986583072570353.dll:JNA native library:6.1.2:Java(TM) Native Access (JNA)
		jsvml.dll:OpenJDK Platform binary:17.0.3.0:Microsoft
		jvm.dll:OpenJDK 64-Bit server VM:17.0.3.0:Microsoft
		kernel.appcore.dll:AppModel API Host:10.0.19041.546 (WinBuild.160101.0800):Microsoft Corporation
		lwjgl.dll
		management.dll:OpenJDK Platform binary:17.0.3.0:Microsoft
		management_ext.dll:OpenJDK Platform binary:17.0.3.0:Microsoft
		mdnsNSP.dll:Bonjour Namespace Provider:3,1,0,1:Apple Inc.
		msvcp140.dll:Microsoft® C Runtime Library:14.29.30139.0 built by: vcwrkspc:Microsoft Corporation
		msvcp_win.dll:Microsoft® C Runtime Library:10.0.19041.789 (WinBuild.160101.0800):Microsoft Corporation
		msvcrt.dll:Windows NT CRT DLL:7.0.19041.546 (WinBuild.160101.0800):Microsoft Corporation
		mswsock.dll:Расширение поставщика службы API Microsoft Windows Sockets 2.0:10.0.19041.2604 (WinBuild.160101.0800):Microsoft Corporation
		napinsp.dll:Поставщик оболочки совместимости для имен электронной почты:10.0.19041.2604 (WinBuild.160101.0800):Microsoft Corporation
		ncrypt.dll:Маршрутизатор Windows NCrypt:10.0.19041.2604 (WinBuild.160101.0800):Microsoft Corporation
		net.dll:OpenJDK Platform binary:17.0.3.0:Microsoft
		nio.dll:OpenJDK Platform binary:17.0.3.0:Microsoft
		ntdll.dll:Системная библиотека NT:10.0.19041.804 (WinBuild.160101.0800):Microsoft Corporation
		perfos.dll:Библиотека объектов производительности системы Windows:10.0.19041.2604 (WinBuild.160101.0800):Microsoft Corporation
		pnrpnsp.dll:Поставщик пространства имен PNRP:10.0.19041.2604 (WinBuild.160101.0800):Microsoft Corporation
		profapi.dll:User Profile Basic API:10.0.19041.844 (WinBuild.160101.0800):Microsoft Corporation
		rasadhlp.dll:Remote Access AutoDial Helper:10.0.19041.546 (WinBuild.160101.0800):Microsoft Corporation
		rsaenh.dll:Microsoft Enhanced Cryptographic Provider:10.0.19041.1 (WinBuild.160101.0800):Microsoft Corporation
		sechost.dll:Host for SCM/SDDL/LSA Lookup APIs:10.0.19041.1 (WinBuild.160101.0800):Microsoft Corporation
		shlwapi.dll:Библиотека небольших программ оболочки:10.0.19041.2604 (WinBuild.160101.0800):Microsoft Corporation
		sunmscapi.dll:OpenJDK Platform binary:17.0.3.0:Microsoft
		ucrtbase.dll:Microsoft® C Runtime Library:10.0.19041.789 (WinBuild.160101.0800):Microsoft Corporation
		vcruntime140_1.dll:Microsoft® C Runtime Library:14.29.30139.0 built by: vcwrkspc:Microsoft Corporation
		verify.dll:OpenJDK Platform binary:17.0.3.0:Microsoft
		win32u.dll:Win32u:10.0.19041.867 (WinBuild.160101.0800):Microsoft Corporation
		windows.storage.dll:API хранения Microsoft WinRT:10.0.19041.2604 (WinBuild.160101.0800):Microsoft Corporation
		winrnr.dll:LDAP RnR Provider DLL:10.0.19041.546 (WinBuild.160101.0800):Microsoft Corporation
		wshbth.dll:Windows Sockets Helper DLL:10.0.19041.546 (WinBuild.160101.0800):Microsoft Corporation
		zip.dll:OpenJDK Platform binary:17.0.3.0:Microsoft
Stacktrace:
	at net.minecraft.client.main.Main.method_44604(Main.java:205)
	at net.minecraft.client.main.Main.main(Main.java:51)
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:462)
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:74)
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23)

-- System Details --
Details:
	Minecraft Version: 1.19.2
	Minecraft Version ID: 1.19.2
	Operating System: Windows 10 (amd64) version 10.0
	Java Version: 17.0.3, Microsoft
	Java VM Version: OpenJDK 64-Bit Server VM (mixed mode), Microsoft
	Memory: 2501072576 bytes (2385 MiB) / 3120562176 bytes (2976 MiB) up to 8589934592 bytes (8192 MiB)
	CPUs: 4
	Processor Vendor: GenuineIntel
	Processor Name: Intel(R) Core(TM) i5-7500 CPU @ 3.40GHz
	Identifier: Intel64 Family 6 Model 158 Stepping 9
	Microarchitecture: Kaby Lake
	Frequency (GHz): 3.41
	Number of physical packages: 1
	Number of physical CPUs: 4
	Number of logical CPUs: 4
	Graphics card #0 name: NVIDIA GeForce GTX 1060 6GB
	Graphics card #0 vendor: NVIDIA (0x10de)
	Graphics card #0 VRAM (MB): 4095.00
	Graphics card #0 deviceId: 0x1c03
	Graphics card #0 versionInfo: DriverVersion=31.0.15.2698
	Graphics card #1 name: Intel(R) HD Graphics 630
	Graphics card #1 vendor: Intel Corporation (0x8086)
	Graphics card #1 VRAM (MB): 1024.00
	Graphics card #1 deviceId: 0x5912
	Graphics card #1 versionInfo: DriverVersion=31.0.101.2112
	Memory slot #0 capacity (MB): 8192.00
	Memory slot #0 clockSpeed (GHz): 2.13
	Memory slot #0 type: DDR4
	Memory slot #1 capacity (MB): 8192.00
	Memory slot #1 clockSpeed (GHz): 2.13
	Memory slot #1 type: DDR4
	Virtual memory max (MB): 25499.29
	Virtual memory used (MB): 12964.83
	Swap memory total (MB): 9216.00
	Swap memory used (MB): 53.89
	JVM Flags: 9 total; -XX:HeapDumpPath=MojangTricksIntelDriversForPerformance_javaw.exe_minecraft.exe.heapdump -Xss1M -Xmx8G -XX:+UnlockExperimentalVMOptions -XX:+UseG1GC -XX:G1NewSizePercent=20 -XX:G1ReservePercent=20 -XX:MaxGCPauseMillis=50 -XX:G1HeapRegionSize=32M
	Fabric Mods: 
		appleskin: AppleSkin 2.4.1+mc1.19
		betterf3: BetterF3 1.4.0
			cloth-config: Cloth Config v8 8.2.88
				cloth-basic-math: cloth-basic-math 0.6.1
		chat_heads: Chat Heads 0.10.5
		dynamicfps: Dynamic FPS 2.2.0
			com_moandjiezana_toml_toml4j: toml4j 0.7.2
		elementa: Elementa 578
		emotecraft: Emotecraft 2.2.7-b.build.47
			bendy-lib: Bendy api 2.1.3-test2
		essential: Essential 12168+deploy-staging+g5c9691d52
		essential-container: essential-container 1.0.0
			essential-loader: essential-loader 1.0.0
		fabric-api: Fabric API 0.75.0+1.19.2
			fabric-api-base: Fabric API Base 0.4.15+8f4e8eb390
			fabric-api-lookup-api-v1: Fabric API Lookup API (v1) 1.6.14+93d8cb8290
			fabric-biome-api-v1: Fabric Biome API (v1) 9.1.0+82db024a90
			fabric-block-api-v1: Fabric Block API (v1) 1.0.2+e415d50e90
			fabric-blockrenderlayer-v1: Fabric BlockRenderLayer Registration (v1) 1.1.24+c6af733c90
			fabric-client-tags-api-v1: Fabric Client Tags 1.0.5+b35fea8390
			fabric-command-api-v1: Fabric Command API (v1) 1.2.16+f71b366f90
			fabric-command-api-v2: Fabric Command API (v2) 2.2.1+413cbbc790
			fabric-commands-v0: Fabric Commands (v0) 0.2.33+df3654b390
			fabric-containers-v0: Fabric Containers (v0) 0.1.41+df3654b390
			fabric-content-registries-v0: Fabric Content Registries (v0) 3.5.1+7c6cd14d90
			fabric-convention-tags-v1: Fabric Convention Tags 1.2.2+27265b3490
			fabric-crash-report-info-v1: Fabric Crash Report Info (v1) 0.2.8+aeb40ebe90
			fabric-data-generation-api-v1: Fabric Data Generation API (v1) 5.3.7+413cbbc790
			fabric-dimensions-v1: Fabric Dimensions API (v1) 2.1.35+0d0f210290
			fabric-entity-events-v1: Fabric Entity Events (v1) 1.5.4+9244241690
			fabric-events-interaction-v0: Fabric Events Interaction (v0) 0.4.34+562bff6e90
			fabric-events-lifecycle-v0: Fabric Events Lifecycle (v0) 0.2.35+df3654b390
			fabric-game-rule-api-v1: Fabric Game Rule API (v1) 1.0.24+b6b6abb490
			fabric-item-api-v1: Fabric Item API (v1) 1.6.5+dce4603e90
			fabric-item-groups-v0: Fabric Item Groups (v0) 0.3.38+9244241690
			fabric-key-binding-api-v1: Fabric Key Binding API (v1) 1.0.25+5c4fce2890
			fabric-keybindings-v0: Fabric Key Bindings (v0) 0.2.23+df3654b390
			fabric-lifecycle-events-v1: Fabric Lifecycle Events (v1) 2.2.4+1b46dc7890
			fabric-loot-api-v2: Fabric Loot API (v2) 1.1.12+83a8659290
			fabric-loot-tables-v1: Fabric Loot Tables (v1) 1.1.15+9e7660c690
			fabric-message-api-v1: Fabric Message API (v1) 5.0.7+93d8cb8290
			fabric-mining-level-api-v1: Fabric Mining Level API (v1) 2.1.23+33fbc73890
			fabric-models-v0: Fabric Models (v0) 0.3.21+c6af733c90
			fabric-networking-api-v1: Fabric Networking API (v1) 1.2.11+10eb22f490
			fabric-networking-v0: Fabric Networking (v0) 0.3.28+df3654b390
			fabric-object-builder-api-v1: Fabric Object Builder API (v1) 4.2.1+d8ef690890
			fabric-particles-v1: Fabric Particles (v1) 1.0.14+4d0d570390
			fabric-recipe-api-v1: Fabric Recipe API (v1) 1.0.1+413cbbc790
			fabric-registry-sync-v0: Fabric Registry Sync (v0) 0.9.32+9244241690
			fabric-renderer-api-v1: Fabric Renderer API (v1) 1.2.1+1adbf27790
			fabric-renderer-indigo: Fabric Renderer - Indigo 0.8.0+1adbf27790
			fabric-renderer-registries-v1: Fabric Renderer Registries (v1) 3.2.24+df3654b390
			fabric-rendering-data-attachment-v1: Fabric Rendering Data Attachment (v1) 0.3.19+6e0787e690
			fabric-rendering-fluids-v1: Fabric Rendering Fluids (v1) 3.0.11+4d0d570390
			fabric-rendering-v0: Fabric Rendering (v0) 1.1.27+df3654b390
			fabric-rendering-v1: Fabric Rendering (v1) 1.12.1+d8ef690890
			fabric-resource-conditions-api-v1: Fabric Resource Conditions API (v1) 2.1.2+aae9039d90
			fabric-resource-loader-v0: Fabric Resource Loader (v0) 0.8.3+251d254a90
			fabric-screen-api-v1: Fabric Screen API (v1) 1.0.32+4d0d570390
			fabric-screen-handler-api-v1: Fabric Screen Handler API (v1) 1.3.7+1cc24b1b90
			fabric-sound-api-v1: Fabric Sound API (v1) 1.0.2+c4f28df590
			fabric-textures-v0: Fabric Textures (v0) 1.0.24+aeb40ebe90
			fabric-transfer-api-v1: Fabric Transfer API (v1) 2.1.6+413cbbc790
			fabric-transitive-access-wideners-v1: Fabric Transitive Access Wideners (v1) 1.3.3+08b73de490
		fabric-language-kotlin: Fabric Language Kotlin 1.8.2+kotlin.1.7.10
		fabricloader: Fabric Loader 0.14.19
		java: OpenJDK 64-Bit Server VM 17
		litematica: Litematica 0.12.5
		malilib: MaLiLib 0.13.0
		minecraft: Minecraft 1.19.2
		modmenu: Mod Menu 4.2.0-beta.2
		optifabric: OptiFabric 1.13.16
			mm: Manningham Mills 2.3
		org_jetbrains_annotations: annotations 13.0
		org_jetbrains_kotlin_kotlin-reflect: kotlin-reflect 1.7.10
		org_jetbrains_kotlin_kotlin-stdlib: kotlin-stdlib 1.7.10
		org_jetbrains_kotlin_kotlin-stdlib-jdk7: kotlin-stdlib-jdk7 1.7.10
		org_jetbrains_kotlin_kotlin-stdlib-jdk8: kotlin-stdlib-jdk8 1.7.10
		org_jetbrains_kotlinx_atomicfu-jvm: atomicfu-jvm 0.18.2
		org_jetbrains_kotlinx_kotlinx-coroutines-core-jvm: kotlinx-coroutines-core-jvm 1.6.4
		org_jetbrains_kotlinx_kotlinx-coroutines-jdk8: kotlinx-coroutines-jdk8 1.6.4
		org_jetbrains_kotlinx_kotlinx-datetime-jvm: kotlinx-datetime-jvm 0.4.0
		org_jetbrains_kotlinx_kotlinx-serialization-cbor-jvm: kotlinx-serialization-cbor-jvm 1.3.3
		org_jetbrains_kotlinx_kotlinx-serialization-core-jvm: kotlinx-serialization-core-jvm 1.3.3
		org_jetbrains_kotlinx_kotlinx-serialization-json-jvm: kotlinx-serialization-json-jvm 1.3.3
		plasmo_voice: Plasmo Voice 1.2.19
		player-animator: Player Animator 0.4.0
		replaymod: Replay Mod 1.19.2-2.6.10
		shulkerboxtooltip: Shulker Box Tooltip 3.2.2+1.19.2
		universalcraft: UniversalCraft 262
		vigilance: Vigilance 278
	Launched Version: fabric-loader-0.14.19-1.19.2
	Backend library: LWJGL version 3.3.1 SNAPSHOT
	Backend API: Unknown
	Window size: <not initialized>
	GL Caps: Using framebuffer using OpenGL 3.2
	GL debug messages: <disabled>
	Using VBOs: Yes
	Is Modded: Definitely; Client brand changed to 'fabric'
	Type: Client (map_client.txt)
	CPU: <unknown>
	OptiFine Version: OptiFine_1.19.2_HD_U_I1
	OptiFine Build: 20221213-150857
	Shaders: null
	OpenGlVersion: null
	OpenGlRenderer: null
	OpenGlVendor: null
	CpuCount: 0

error is code -1

**Comment by warjort** [comment, marked as answer]
`Caused by: java.lang.NullPointerException: Cannot read field "emotesDir" because "io.github.kosmx.emotes.executor.EmoteInstance.config" is null
at io.github.kosmx.emotes.executor.EmoteInstance.getExternalEmoteDir(EmoteInstance.java:32)`

Some issue with the emotescraft configuration file. Contact the mod author.

**Reply by Sleeper77713** [reply]
How can i do this?

**Reply by warjort** [reply]
Use the "issues" tab on curseforge for the mod. 
https://www.curseforge.com/minecraft/mc-mods/emotecraft
